gspread/models.py

```diff
@@ -149,6 +149,22 @@ def batch_update(self, body):
 
         return r.json()
 
+    def notes_get(self, range):
```

Comment:
I'm not convinced that there is a need for this public method in the `Spreadsheet` class. It'd be great to have as minimal api surface area as possible and I actually regret that I added other convenience methods as public methods in `Spreadsheet`.

Let me know if you have a specific reason for this public method. If you don't, please consider moving this code to `Worksheet.get_note`.

Reply:
I just added that method for consistency with values_get(). I’ve removed it and transferred the logic to worksheet.get_note()